src/main/scala/group/scala/karazin/circe/literal/extras/macros.scala

```diff
@@ -17,6 +17,7 @@ object macros:
     private val IntUnit = 0
     private val StringUnit = ""
     private val JsonObjectUnit = JsonObject.empty
+    private val ShortUnit = 0
```

Comment:
I guess it is better to use one unit for all types of numbers.
That is, use `val NumberUnit = 0` as IntUnit, ShortUnit, LongUnit, etc